lib/mi_bridges/driver/others_types_of_income_page.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class OtherTypesofIncomePage < BasePage
+      TITLE = "Other Types of Income"
+
+      def setup; end
+
+      def fill_in_required_fields
+        # social security?
+        # foster care / adoption
+        check_unemployment
+        check_disability
+        check_workers_compensation
+        check_pension_or_retirement
+        check_other_income
+      end
+
+      def continue
+        click_on "Next"
+      end
+
+      private
+
+      def check_unemployment
+        check_in_section(
+          section_name,
+          condition: additional_income?("unemployment_insurance"),
+          for_label: "Unemployment Benefits",
+        )
+      end
+
+      def check_disability
+        check_in_section(
+          section_name,
+          condition: additional_income?("ssi_or_disability"),
+          for_label: "Disability Benefits",
+        )
+      end
+
+      def check_workers_compensation
+        check_in_section(
+          section_name,
+          condition: additional_income?("workers_compensation"),
+          for_label: "Workers Compensation",
+        )
+      end
+
+      def check_pension_or_retirement
+        check_in_section(
+          section_name,
+          condition: additional_income?("pension"),
+          for_label: "Pension or Retirement",
+        )
+      end
+
+      def check_other_income
+        check_in_section(
+          section_name,
+          condition: additional_income?("other"),
+          for_label: "Other Income",
+        )
+      end
+
+      def section_name
+        "star#{first_name_section(primary_member).capitalize}sIncomeInformation"
```

Comment:
Maybe consider extracting something like:

```
def pluralized_first_name_of_primary_member
  first_name_section(primary_member).capitalize
end
```
?

Seems like a lot of activity for an interpolated string 🎞